webpack/assets/javascripts/react_app/redux/actions/statistics/index.js

```diff
@@ -0,0 +1,18 @@
+import API from '../../../API';
+import {
+  STATISTICS_DATA_REQUEST,
+  STATISTICS_DATA_SUCCESS,
+  STATISTICS_DATA_FAILURE
+} from '../../consts';
+
```

Comment:
is this a duplicate file? whats the diff between actions/statistics and actions/statistics/index.js?